lib/rake.rb

```diff
@@ -21,7 +21,17 @@
 # IN THE SOFTWARE.
 #++
 
-module Rake; end
+module Rake
+  module AllExceptionsExceptOnesWeMustNotRescue
```

Comment:
I wasn't sure where would be a good place for this, just put it here for a draft PR.